packages/forma-36-react-components/src/components/DateTime/DateTime/DateTime.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, text, date } from '@storybook/addon-knobs';
+
+import DateTime from './DateTime';
+import { DateTimeFormat } from '../dateUtils';
+
+const formatOptions: DateTimeFormat[] = [
+  'FULL',
+  'DATE_ONLY',
+  'TIME_ONLY',
+  'WEEKDAY_DATE',
+];
+
+const defaultDate = new Date('2019-08-12T08:00:00Z');
+
+storiesOf('Components/DateTime', module)
```

Comment:
@freakyfelt thx a lot for this contribution! 🎉
As we recently bumped storybook to the new version, it is possible now to use new format for the story. We would slowly refactor stories into that. Maybe you would like to use the new one?
More info around this https://storybook.js.org/docs/react/api/csf/
We have couple of components in the new format, if you would like to have a look https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/Accordion/Accordion.stories.tsx